robert_gazebo/worlds/ucu.world

```diff
@@ -9165,6 +9165,410 @@
       <static>1</static>
       <allow_auto_disable>1</allow_auto_disable>
       <pose frame=''>-1.2423 -1.3637 0 0 -0 0</pose>
+    </model>
```

Comment:
Why did You changed the ucu.world file?

Reply:
added columns in lectorium

Reply:
The columnt were already added here https://github.com/ucuapps/robert/pull/54 weren't they?

Reply:
I add them to this branch before #54 was approved.